app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ForgotPasswordsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
For fetching a single record, using `find_by` is more efficient.  Check notes at the bottom of this page - https://www.gotealeaf.com/lessons/a3a6fbb9/assignments/1566